src/main.ts

```diff
@@ -1,5 +1,6 @@
 import '@babel/polyfill'
 import Vue from 'vue'
+import BootstrapVue from 'bootstrap-vue'
```

Comment:
Maybe a bit overkill for only a popup, but we might use it in other situations.

Reply:
It does quite a bit of hairy details work for us too, I believe.